.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+*.jld
+Manifest.toml
```

Comment:
Should manifest be ignored or committed? If it's like package management in a few other languages it seems like something that should be kept to ensure that git-shas are maintained.